components/core/src/Core.js

```diff
@@ -2,14 +2,17 @@
 
 import React, { useContext } from 'react';
 import { Global, jsx } from '@emotion/core';
+import merge from 'lodash.merge';
 import { useTheme, UserModeContext } from './Theme';
 import { paint } from './utils';
 
-import merge from 'lodash.merge';
-
 export const Core = ({ children }) => {
-	const { colors, breakpoints, font: styleFont, typography } = useTheme();
-	const isKeyboardUser = useContext(UserModeContext);
+	const {
+		COLORS,
+		LAYOUT: { breakpoints },
+		font: styleFont,
```

Comment:
I'd say just keep as is with uppercase for the token categories. (I forgot to remove that font destructure since it doesnt exist in the new tokens :man_facepalming:)